resources/test/conftest.py

```diff
@@ -17,8 +17,10 @@ def pytest_addoption(parser):
     parser.addoption("--binary", action="store", default=None, help="path to browser binary")
 
 def pytest_collect_file(path, parent):
+    test_type = os.path.relpath(str(path), HERE).split(os.path.sep)[1]
+
     if path.ext.lower() == '.html':
-        return HTMLItem(str(path), parent)
+        return HTMLItem(str(path), test_type, parent)
```

Comment:
Nit: only compute `test_type` if you need it.

Reply:
Sure